torchvision/transforms/functional_tensor.py

```diff
@@ -940,8 +940,8 @@ def erase(img: Tensor, i: int, j: int, h: int, w: int, v: Tensor, inplace: bool
     return img
 
 
-def _create_identity_grid(size: List[int]) -> Tensor:
-    hw_space = [torch.linspace((-s + 1) / s, (s - 1) / s, s) for s in size]
+def _create_identity_grid(size: List[int], device: torch.device) -> Tensor:
```

Comment:
I thought to do this directly on V1 to avoid copy-pasting it.

Reply:
We can do better than this code, for example: https://github.com/pytorch/vision/pull/6897/files#diff-8d715258c77dea35427ef8b4605c5dca131717b7e2f6ca1ee3afa99392813105R1050-R1059

Reply:
Perfect, reverting this one.